postgres/tasks.py

```diff
@@ -44,7 +45,7 @@ def import_snapshot(ctx, snapshot_name=None, migrate=True):
     # Make minor adjustments to dumps for legacy dumps to work on AWS
     if migrate:
         print("Migrating dump file")
-        ctx.run(f"sed -i 's/OWNER TO surf/OWNER TO postgres/g' {snapshot_file_path}", echo=True)
+        ctx.run(f"sed -i '' 's/OWNER TO surf/OWNER TO postgres/g' {snapshot_file_path}", echo=True)
```

Comment:
I'm not sure what this does. ``echo '' 'hello world'`` will simply put a space in front of hello world. How will sed parse this exactly?

Reply:
Ok, I think I understand. This is then an empty prefix? How will that then affect the new file if backup is exactly the same as new file?

Reply:
https://stackoverflow.com/questions/16745988/sed-command-with-i-option-in-place-editing-works-fine-on-ubuntu-but-not-mac

MacOS somehow requires a backup-file extension for `-i`. So it will take the regex as extension and the command fails. With `-i ''` it will not write to a backup and instead just replace the file during runtime.

I researched a bit more and `-i.bak` works on both GNU and BSD (macOS) systems. It writes to a temporary `.bak` file and replaces the original when the command succeeds.

So I updated it 😉